net/process.go

```diff
@@ -18,26 +18,26 @@ import (
 	"fmt"
 	"sync"
 
-	dag "github.com/ipfs/boxo/ipld/merkledag"
-	blocks "github.com/ipfs/go-block-format"
-	"github.com/ipfs/go-cid"
-	ipld "github.com/ipfs/go-ipld-format"
+	"github.com/ipfs/boxo/fetcher"
+	"github.com/ipld/go-ipld-prime"
+	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
 	"github.com/sourcenetwork/corelog"
 
 	"github.com/sourcenetwork/defradb/client"
 	"github.com/sourcenetwork/defradb/datastore"
 	"github.com/sourcenetwork/defradb/errors"
 	"github.com/sourcenetwork/defradb/internal/core"
+	coreblock "github.com/sourcenetwork/defradb/internal/core/block"
 	"github.com/sourcenetwork/defradb/internal/db/base"
 	merklecrdt "github.com/sourcenetwork/defradb/internal/merkle/crdt"
 )
 
 type blockProcessor struct {
 	*Peer
-	txn    datastore.Txn
-	col    client.Collection
-	dsKey  core.DataStoreKey
-	getter ipld.NodeGetter
+	txn     datastore.Txn
+	col     client.Collection
+	dsKey   core.DataStoreKey
+	fetcher fetcher.Fetcher
```

Comment:
suggestion: Similar to my import alias suggestion, I suggest not calling this property `fetcher` - it has a very strong association with out database fetchers and confused me for a while when reading.

Reply:
✅